File: QKit/QKit/src/android/file_tools.cpp

#include <QAndroidJniEnvironment>
#include <QAndroidJniObject>
#include <QtAndroid>
#include <jni.h>

QString android_getStandardPicturePath()
{
    QAndroidJniObject  ret = QAndroidJniObject::callStaticObjectMethod("qkit/FileTools","standardPicturePath","()Ljava/lang/String;");

    return ret.toString();
}

File: QKit/QKit/src/android/java/qkit/FileTools.java
package qkit;

import java.lang.String;

import android.os.Environment;
import android.util.Log;

class FileTools
{

    public static String standardPicturePath()
    { 	String ret = Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DCIM).getAbsolutePath();
    	Log.d("standardPicturePath:", ret);
        return ret;
    }

};

